internal/image/source.go

```diff
@@ -161,6 +164,15 @@ func NewSource(config Config, migrations embed.FS, migrationsThumbs embed.FS) *S
 	source.database = NewDatabase(filepath.Join(config.DataDir, "photofield.cache.db"), migrations)
 	source.imageInfoCache = newInfoCache()
 	source.pathCache = newPathCache()
+	
+	r, err := rgeo.New(rgeo.Provinces10, rgeo.Cities10)
```

Comment:
This takes about half a minute to init on my (admittedly quite old) PC.

Maybe a good idea would be to load it async (e.g. in a goroutine) just so that restarts don't take that long?

Alternatively `rgeo` could be extended to use a sqlite file instead of JSON for the loading, or maybe even for the queries itself. Should be way faster :)